app/Livewire/Admin/UserManager.php

```
@@ -0,0 +1,91 @@
+                        ->orWhere('phone', 'like', '%' . $this->search . '%');
+                });
+            })
+            ->when($this->statusFilter !== '', function ($query) {
+                $query->where('status', $this->statusFilter);
+            })
+            ->when($this->emailVerifiedFilter !== '', function ($query) {
+                if ($this->emailVerifiedFilter == '1') {
+                    $query->whereNotNull('email_verify');
+                } else {
+                    $query->whereNull('email_verify');
+                }
```

PHPMD (2.15.0)

The method render uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)